tsfile/src/main/java/org/apache/iotdb/tsfile/fileSystem/fileInputFactory/HDFSInputFactory.java

```diff
@@ -35,19 +37,16 @@
       Class<?> clazz = Class.forName("org.apache.iotdb.hadoop.fileSystem.HDFSInput");
       constructor = clazz.getConstructor(String.class);
     } catch (ClassNotFoundException | NoSuchMethodException e) {
-      logger.error(
-          "Failed to get HDFSInput in Hadoop file system. Please check your dependency of Hadoop module.",
-          e);
+      logger.error("Failed to get HDFSInput in Hadoop file system. Please check your dependency of Hadoop module.", e);
```

Comment:
I think you need to use idea to format the codes......... Maybe this line exceeds the max length.